motoman_sia50_support/urdf/sia50d_macro.xacro

```diff
@@ -0,0 +1,209 @@
+            <limit lower="-3.1415" upper="3.1415" effort="18.5" velocity="2.967051" />
+        </joint>
+        <joint name="${prefix}joint_l" type="revolute">
+            <parent link="${prefix}link_s"/>
+            <child link="${prefix}link_l"/>
+            <origin xyz="0.145 0 0.228" rpy="0 0 0"/>
+            <axis xyz="0 1 0" />
+            <limit lower="-1.047198" upper="2.181662" effort="28.4" velocity="2.268928" />
+            <!-- 5883.99 -->
+        </joint>
+        <joint name="${prefix}joint_e" type="revolute">
+            <parent link="${prefix}link_l"/>
+            <child link="${prefix}link_e"/>
+            <origin xyz="0 0 0.875" rpy="0 0 0"/>
+            <axis xyz="0 0 1" />
+            <limit lower="-2.967064" upper="2.967064" effort="22" velocity="2.26893" />
+            <!-- 2549.73 -->
+        </joint>
+        <joint name="${prefix}joint_u" type="revolute">
+            <parent link="${prefix}link_e"/>
+            <child link="${prefix}link_u"/>
+            <origin xyz="0 0 0" rpy="0 0 0"/>
+            <axis xyz="0 -1 0" />
+            <limit lower="-0.610869" upper="3.752462" effort="22" velocity="2.26893" />
+            <!-- 2549.73 -->
+        </joint>
+        <joint name="${prefix}joint_r" type="revolute">
+            <parent link="${prefix}link_u"/>
+            <child link="${prefix}link_r"/>
+            <origin xyz="0.61 0 0" rpy="0 0 0"/>
+            <axis xyz="-1 0 0" />
+            <limit lower="-2.96706" upper="2.96706" effort="6" velocity="2.268928" />
+            <!-- 720.2 -->
+        </joint>
+        <joint name="${prefix}joint_b" type="revolute">
+            <parent link="${prefix}link_r"/>
+            <child link="${prefix}link_b"/>
+            <origin xyz="0 0 0" rpy="0 0 0"/>
+            <axis xyz="0 -1 0" />
+            <limit lower="-2.181662" upper="2.181662" effort="6" velocity="2.268928" />
+            <!-- 720.2 -->
+        </joint>
+        <joint name="${prefix}joint_t" type="revolute">
+            <parent link="${prefix}link_b"/>
+            <child link="${prefix}link_t"/>
+            <origin xyz="0.35 0 0" rpy="0 0 0"/>
+            <axis xyz="-1 0 0" />
+            <limit lower="-3.141593" upper="3.141593" effort="2.5" velocity="3.490659" />
+            <!-- 255.76 -->
+        </joint>
+        <joint name="${prefix}link_t-tool0" type="fixed">
+            <origin xyz="0 0 0.0" rpy="0 0 -3.1415926535"/>
+            <parent link="${prefix}link_t" />
+            <child link="${prefix}tool0" />
+        </joint>
+        <!-- end of joint list -->
+
+        <!-- ROS base_link (via base_link) to Motoman Robot (not Base) Frame transform -->
+        <!-- <link name="${prefix}base" />
+        <joint name="${prefix}base_link-base" type="fixed">
+            <origin xyz="0 0 0.410" rpy="0 0 0"/>
+            <parent link="${prefix}base_link"/>
+            <child link="${prefix}base"/>
+        </joint> -->
```

Comment:
Could you clarify why this is commented?

We should not merge any more macros without the `base` and `tool0` frames defined.